### ISSUE TITLE
Clock keeps running when Windows is "resting"

### ISSUE THREAD

**Issue by riemaxi** [Gecode/gecode#16]
On Windows, when the process gets suspended the clock (when inheriting from Script) keeps counting.

The following is the report from an experiment with 10 runs:
run    failures                   size                     runtime
================================
1       1527988                 123                     60022.0
2       1525638                 123                     60022.0
3       1524347                 123                     60022.0
4       1525846                 123                     60022.0
5       1526807                 123                     60023.0
6       1524908                 123                     60022.0
7       1507045                 123                     60023.0
8       1525250                 123                     60021.0
9       1525695                 123                     60021.0
10      584070                  123                     31298851.0

I had this running all night. When I got up the process was suspended on the run 10.
Check out the difference between the runtime of run 10 and the rest.

**Comment by riemaxi** [comment]
Nope, it does not ... if you do not do anything about it .... is your
problem ...

//Sam
Have you ever considered that Earth might be a spaceship?

**Comment by guidotack** [comment]
The clock in Driver is wall time, not CPU time. If you need CPU time you will have to implement your own timing code.

**Comment by riemaxi** [comment]
Hi Guido
I shouid not have to implement another clock, I am satisfied with that one
... it is just that there is a bug in the Script class, which is the one
holding it.

Guys, I suggest you catch that exception and save the time up to that
point, when windows has got back from suspension, then show the current cpu
time + the difference between the restore time point and the exception time
point. I believe that would be a workaround ... consequences? ... I do not
know, but I can run some tests.

By the way, talking abut saving space + stats info in case of unexpected
interruption. Do you have fail-restore mechanism implemented or you have to
relay on some platform that implement it?

regards
//Sam
Have you ever considered that Earth might be a spaceship?

**Comment by guidotack** [comment]
If this is important for you and you know how to implement it, create a pull request and we may include it.
